panacea/query_client.go

```diff
@@ -112,6 +123,30 @@ func NewQueryClient(ctx context.Context, config *config.Config, info TrustedBloc
 	}, nil
 }
 
+// refresh update light block, when the last light block has been updated more than trustPeriod * 2/3.
+func refresh(ctx context.Context, lc *light.Client, trustPeriod time.Duration) error {
+	logrus.Info("check latest light block")
+	lastBlockHeight, err := lc.LastTrustedHeight()
+	if err != nil {
+		return err
+	}
+	lastBlock, err := lc.TrustedLightBlock(lastBlockHeight)
+	if err != nil {
+		return err
+	}
+	lastBlockTime := lastBlock.Time
+	currentTime := time.Now()
+	timeDiff := currentTime.Sub(lastBlockTime)
+	if timeDiff > trustPeriod*2/3 {
```

Comment:
Should this `timeDiff` be less than `trustedPeriod`?
If the `timeDiff` is greater than `trustedPeriod`, we need to handle differently (maybe skipping verification?)

Reply:
Yes, you're right. If `timeDiff` is greater than `trustedPeriod`, it returns error when `Update` (exactly,  expired error occurs in the `verify`). 
It looks like we can solve it, but I think the logic is complicated, and I'm not sure if that logic is needed yet. 
Let's talk together.